gems/quilt_rails/lib/quilt_rails/monkey_patches/active_support_reloader.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
```

Comment:
A gem monkeypatching Rails is 💩 😞  If anyone has a better suggestion, I'm eager to hear it.